prov/gni/src/gnix_mr_cache.c

```diff
@@ -81,20 +85,19 @@ typedef struct gnix_mr_cache_entry {
 } gnix_mr_cache_entry_t;
 
 /* forward declarations */
-int _gnix_mr_cache_init(
-		gnix_mr_cache_t      **cache,
-		gnix_mr_cache_attr_t *attr);
```

Comment:
FYI, removed this forward declaration, since it is not needed (already in the .h).